plugin/al/usdtransaction/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(PY_INIT_FILES
 # copy to build location for module tests
 foreach(INPUT_FILE ${PY_INIT_FILES})
   string(REPLACE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR} OUTPUT_FILE ${INPUT_FILE})
-  execute_process(COMMAND ${CMAKE_COMMAND} -E copy ${INPUT_FILE} ${OUTPUT_FILE})
+  execute_process(COMMAND "${CMAKE_COMMAND}" -E copy "${INPUT_FILE}" "${OUTPUT_FILE}")
```

Comment:
wrap all the full path variables around quotes in case there are located in weird places with whitespaces and special characters on Windows. Just to be safe!